README.md

```diff
@@ -121,9 +121,8 @@ The following image is a good illustration of a template.
 **basePdf** can be given a `string`(base64), `ArrayBuffer`, or `Uint8Array`.  
 A blank A4 PDF can be imported with `BLANK_PDF`. You can use it to check how it works.
 
-**schemas** can only utilize text by default, but you can load images and various barcodes like QR codes as plugins from the `@pdfme/schemas` package.  
-Additionally, you can create your own schemas, allowing you to render types other than the ones mentioned above.  
-Check detail about [Custom Schemas](https://pdfme.com/docs/custom-schemas) from here
+**Schemas** can only utilize text by default, but you can load images and various barcodes like QR codes as plugins from the `@pdfme/schemas` package.  
+Additionally, you can create your own schemas, allowing you to render types other than the ones mentioned above. Check detail about [Custom Schemas](https://pdfme.com/docs/custom-schemas) from here.
```

Comment:
❤️

Reply:
In addition to this root README.md, the README.md files in the common, schemas, ui, and generator packages in this repository also need to be similarly edited.
The README.md for each package is displayed on the npm website. For example: https://www.npmjs.com/package/@pdfme/generator

However, since the content is mostly duplicated, they should perhaps be linked to this repository on Github instead.

In any case, this is not directly related to this PR, so let's address it in a separate PR.

Reply:
no problem, I will update the other README.md files for completeness 👍

Reply:
updated